js/stores/CurrentLocation.js

```diff
@@ -16,14 +16,16 @@ class CurrentLocation extends Events {
       hasGranted: false,
       initialSet: window.location.pathname.split('/')[1] !== 's',
     }
+  }
+
+  async componentDidMount() {
```

Comment:
@consindo how do you feel about this stuff moving out of the constructor - i don't think it should be there anyway

Reply:
agree